test/functional/feature_bip68_sequence.py

```diff
@@ -32,7 +32,7 @@ def set_test_params(self):
         self.extra_args = [
             [
                 "-acceptnonstdtxn=1",
-                "-peertimeout=9999",  # bump because mocktime might cause a disconnect otherwise
```

Comment:
> I think it might make more sense in general to set -peertimeout=9999 for all the test cases (ie, in the framework generated bitcoin.conf)

Changing the behaviour of all tests is outside the scope of this PR, but I'm happy to review any proposal you have for making the tests more robust.